### PR TITLE
fix(nixos-wiki): unthemed search bar

### PR DESCRIPTION
**Pull request by somerand0mcat**
i was doing some testing with the nixos.wiki userstyle and i found out that the search bar wasn't themed so i decided to fix it

image
image

### CODE REVIEW

File: styles/nixos.wiki/catppuccin.user.css [hunk 1]
```diff
@@ -322,6 +322,40 @@
...
+      background-color: @base;
+      border: none;
+    }
+    
+    .search-types .current a {
+      color: @text;
+    }
+    
+    .oo-ui-inputWidget-input {
+       background-color: @subtext1 !important;
```
Comment (isabelroses) [inline styles/nixos.wiki/catppuccin.user.css:336]: we shouldn't make subtext a background color

File: styles/nixos.wiki/catppuccin.user.css [hunk 2]
```diff
@@ -322,6 +322,40 @@
...
+       color: @base !important;
+    }
+
+    & when (@lookup = latte) {
+     .oo-ui-inputWidget-input {
+       background-color: @base !important;
+       color: @text
+      }
+     .oo-ui-buttonElement-button {
+       background-color: @blue !important;
```
Comment (isabelroses) [inline styles/nixos.wiki/catppuccin.user.css:355]: perhaps accent instead

File: styles/nixos.wiki/catppuccin.user.css [hunk 3]
```diff
@@ -322,6 +322,40 @@
...
+       border: none;
```
Comment (isabelroses) [inline styles/nixos.wiki/catppuccin.user.css:356]:
```suggestion
```
we don't want an opinionated change like this.

File: styles/nixos.wiki/catppuccin.user.css [hunk 4]
```diff
@@ -322,6 +322,40 @@
     #footer {
       background-color: @mantle;
     }
+    
+    .mw-search-profile-tabs, {
+      background-color: @base;
+      border: none;
```
Comment (isabelroses) [inline styles/nixos.wiki/catppuccin.user.css:328]:
```suggestion
```
we don't want an opinionated change like this.

File: styles/nixos.wiki/catppuccin.user.css [hunk 5]
```diff
@@ -322,6 +322,40 @@
...
+    }
+    
+    .oo-ui-inputWidget-input {
+       background-color: @subtext1 !important;
+       color: @base;
+    }
+    
+    .oo-ui-buttonElement-button {
+       background-color: @blue !important;
+       border: none;
```
Comment (isabelroses) [inline styles/nixos.wiki/catppuccin.user.css:342]:
```suggestion
```
we don't want an opinionated change like this.

File: styles/nixos.wiki/catppuccin.user.css [hunk 6]
```diff
@@ -322,6 +322,40 @@
...
+     }
+    
+    .oo-ui-labelElement-label {
+       color: @base !important;
+    }
+
+    & when (@lookup = latte) {
```
Comment (isabelroses) [inline styles/nixos.wiki/catppuccin.user.css:349]: try make latte the same as the dark flavours otherwise should be fine...

**Review by isabelroses — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
Seeing as nixos.wiki is no longer the suggested wiki site (see https://github.com/catppuccin/userstyles/issues/781) I'm not inclined to accept fixes for this userstyle. Thoughts @isabelroses?

**Comment by isabelroses** [comment]
> Seeing as nixos.wiki is no longer the suggested wiki site (see #781) I'm not inclined to accept fixes for this userstyle. Thoughts @isabelroses?

I don't see too much harm in accepting fixes but ideally we should push for a new theme

File: styles/nixos.wiki/catppuccin.user.css (hunk 6 above)
Comment (somerand0mcat) [reply styles/nixos.wiki/catppuccin.user.css:347]: the search bar breaks when i switch to latte so

Comment (somerand0mcat) [reply styles/nixos.wiki/catppuccin.user.css:347]: i'll keep it

**Review by isabelroses — CHANGES_REQUESTED** [review]
1) Rember to update the version number

2) These update seems to break some stuff here
image
image

**Comment by somerand0mcat** [comment]
dw i'll fix it

**Comment by somerand0mcat** [comment]
@isabelroses fixed
image

**Comment by uncenter** [comment]
See lint error.

**Comment by somerand0mcat** [comment]
@uncenter fixed

**Comment by somerand0mcat** [comment]
i also added some border colors to make it look pretty good
image

**Review by isabelroses — APPROVED** [review]

**Pull request merged**